Tasks/Inc/Tasks.h

```diff
@@ -106,6 +111,7 @@ extern CPU_STK CheckContactor_Stk[TASK_CHECK_CONTACTOR_STACK_SIZE];
 extern CPU_STK BatteryBalance_Stk[TASK_BATTERY_BALANCE_STACK_SIZE];
 extern CPU_STK CLI_Stk[TASK_CLI_STACK_SIZE];
 extern CPU_STK BLE_Stk[TASK_BLE_STACK_SIZE];
+extern CPU_STK Print_Stk[TASK_BLE_STACK_SIZE];
```

Comment:
no separate define for print stack size?

Reply:
Is this better?